app/enterprise/changelog.md

```diff
@@ -4,6 +4,167 @@ no_search: true
 no_version: true
 skip_read_time: true
 ---
+## 2.3.0.0
+**Release Date** 2020/02/10
+
+### **Kong Gateway (Enterprise)**
+The following sections list {{site.ee_product_name}}-exclusive updates,
+features, and fixes for the **2.3.0.0** version.
+
+Kong Enterprise is renamed to Kong Gateway, or Kong Gateway (Enterprise) going forward.
+
+### Features
+
+#### Enterprise
+- Kong Gateway can now run in free mode, without a license, which gives users access to Kong Manager and an easy upgrade path to Enterprise.
+- A (non-OSS) Kong Gateway running in its default Free Mode can be upgraded to add
```

Comment:
Let's be consistent in capitalization of "free mode" (we use it as a proper name in this line but lower case on the line above).  I think it's fine to use proper naming capitalization (Free Mode) even though we don't really have anything internally that delineates it as such but I'd lean in the direction of not considering it a proper name and instead calling it out as "a free mode" (on the line above) and "default free mode" in this line as non-proper

Reply:
I'd lean towards not having this capitalized, as excessive capitalization is already a problem in the docs and often leads to interrupted flow.

Reply:
fixed to be "free mode"